pages/doc/logging_overview.md

```diff
@@ -3,16 +3,16 @@ title: Get Started with Logs (Beta)
 tags: [getting started, logs]
 sidebar: doc_sidebar
 permalink: logging_overview.html
-summary: Learn about Tanzu Observability metrics, logs, and traces.
+summary: Learn about Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront) metrics, logs, and traces.
```

Comment:
VMware Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront)